code/README.md

```diff
@@ -1,19 +1,24 @@
-# Technigo React Starter App
+# Survey form with React
 
-This app has been generated using `create-react-app`, then cleaned up a little so we have a project with just the elements we need to get started.
+Goal was to build a survey with React with at least 3 question using 3 different types, radio buttons, dropdown, input field and have a submit button that hide the form and show a summary of the inputs. 
 
-Add components in the `src` folder and use them in `src/app.js` to get started.
+## The problem
 
-## Usage
+It has been hard as I do not understand how to combine the components and therefore only used one. 
+Most difficult is to create the summary of the inputs and how to collect the data. 
 
-1. On GitHub, click the 'use this template' button to create a new repo on your account
-1. Clone the new repo to your computer
-1. Install required dependencies with npm: `npm install`
-1. Start the project: `npm start`
+Stackoverflow, a lot of tutorials and read up on React has made it possible to accomplish this form. still missing a bit styling, but functionality is working as it should. 
 
-Alternatively, if you don't want a new git repo on your account;
 
-1. Download & extract the [latest release](https://github.com/Technigo/react-starter/releases/latest) from GitHub
-1. Open the directory in the terminal: `cd /path/to/react-starter`
-1. Install required dependencies with npm: `npm install`
-1. Start the project: `npm start`
+
+## View it live
+
+Link to live site: 
+https://optimistic-stonebraker-71d34e.netlify.app
+
+Feedback 
+
+- How can I separate the Form into components? 
+- how to show one question at the time, best practice to do it? 
```

Comment:
I did it like this:
     1. Define a state to update the current question: const [question, setQuestion] = useState(0)
     2. Function that will update the state:   const onNextQuestionChange = () => {
                                                                         setQuestion(question + 1)
                                                                     }
     3. Create a button that will call this function each time it is clicked: 
    <button onClick={onNextQuestionChange}> Next <button>
     4. Use the conditional for each question: 
     {question === 0 && (
<div className="container-input-fields">
        <div className="name-label">
          <label htmlFor={name}>Number of tickets:</label>
          <input 
            className="name-input" 
            id="name" 
            type="number" 
            value={name} 
            onChange={onNameChange}
          /> 
        </div>
          <button onClick={onNextQuestionChange}> Next <button>)
}